file-echo-api/file-echo-api/Main.hs

```diff
@@ -29,5 +29,6 @@ serverMethods :: [(Text, Argo.MethodType, JSON.Value -> Argo.Method FES.ServerSt
 serverMethods =
   [ ("load",           Argo.Command,   Argo.method FES.loadCmd)
   , ("clear",          Argo.Command,   Argo.method FES.clearCmd)
+  , ("implode",        Argo.Command,   Argo.method FES.implodeCmd)
```

Comment:
Why is this a command, if the server never enters a new state through implosion? (yes, nit-picky)